theseus/geometry/so3.py

```diff
@@ -84,10 +87,10 @@ def _compose_impl(self, so3_2: LieGroup) -> "SO3":
         raise NotImplementedError
 
     def _inverse_impl(self, get_jacobian: bool = False) -> "SO3":
-        raise NotImplementedError
+        return SO3(data=self.data.transpose(1, 2).clone())
 
     def to_matrix(self) -> torch.Tensor:
-        return self.data
+        return self.data.clone()
```

Comment:
Same question regarding `.clone()`

Reply:
Please refer to the last [reply](https://github.com/facebookresearch/theseus/pull/47#discussion_r793064803).